pkg/audit/audit.go

```diff
@@ -47,7 +46,14 @@ func Batch(ctx context.Context, secrets []string, secStore secretGetter) error {
 
 	// Spawn workers that run the auditing of all secrets concurrently.
 	validator := crunchy.NewValidator()
-	maxJobs := runtime.NumCPU()
+
+	// It would be nice to parallelize this operation and limit the maxJobs to
+	// runtime.NumCPU(), but sadly this causes various problems with multiple
+	// gnupg jobs running parallely. See the entire discussion here:
```

Comment:
./pkg/audit/audit.go:52:23: "parallely" is a misspelling of "parallelly" :-)

Reply:
Thank you! I can not count how many times in my life I have typoed this word :smile: